app/src/main/res/values/strings.xml

```diff
@@ -20,6 +20,8 @@
     <string name="unpin_conversation">Unpin</string>
     <string name="forward_message">Forward</string>
     <string name="compress_error">Unable to compress image to selected size</string>
+    <string name="sms">SMS</string>
+    <string name="mms">MMS</string>
```

Comment:
lets move SMS and MMS to donottranslate.xml

Reply:
[done :+1: ](https://github.com/SimpleMobileTools/Simple-SMS-Messenger/pull/363/commits/7695271de35f645b19482c61ae61e893bf9eabdd)